src/main/java/io/github/linoxgh/moretools/Items.java

```diff
@@ -1,32 +1,40 @@
 package io.github.linoxgh.moretools;
 
-import java.util.List;
-
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import me.mrCookieSlime.Slimefun.cscorelib2.config.Config;
 import org.bukkit.ChatColor;
 import org.bukkit.Material;
 import org.bukkit.inventory.meta.ItemMeta;
 
-import me.mrCookieSlime.Slimefun.cscorelib2.config.Config;
-import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import java.util.List;
 
 public class Items {
-    
+
     // TOOLS
-    public static final SlimefunItemStack CRESCENT_HAMMER = new SlimefunItemStack("CRESCENT_HAMMER", Material.IRON_HOE, "&bCrescent Hammer", "&7&oActually this is a wrench, really.", "", "&eLeft Click &7> Dismantles the machine.");
+    public static final SlimefunItemStack CRESCENT_HAMMER = new SlimefunItemStack(
```

Comment:
I prefer keeping this as 1 line, just like in Slimefun. I think it's more readable that way when the item amount gets bigger.

Reply:
coding standards so it would be better to keep it like this

Reply:
> coding standards so it would be better to keep it like this

There is no standard in this. It is all personal. There is no performance improvement.